src/token.cairo

```diff
@@ -1,3 +1,4 @@
 pub mod erc1155;
 pub mod erc20;
+pub mod erc2981;
```

Comment:
Since this is not a different token but an extension to both ERC721 and ERC1155, it makes sense for it to be placed under a common module instead.

Reply:
I have moved the implementation to `src/token/common/erc2981`